graphql/resolvers.go

```diff
@@ -43,8 +45,29 @@ func (r *queryResolver) UserPatches(ctx context.Context, userID string) ([]*mode
 	return patchPointers, nil
 }
 
+func (r *taskResolver) TestResults(ctx context.Context, obj *model.APITask) ([]*model.APITest, error) {
+	tests, err := r.sc.FindTestsByTaskId(*obj.Id, "", "", "", 0, obj.Execution)
```

Comment:
Do we need to paginate this? In slightly unusual situations this can return over 100,000 objects for a single task

Reply:
agree on pagination. when we chatted, @SupaJoon mentioned that he was going to paginate this

Reply:
I was going to make a separate PR for paginating the testresults. 
https://jira.mongodb.org/browse/EVG-7206

Reply:
seems reasonable to do it all in one PR

Reply:
@tgrander @john-m-liu It doesn't seem possible to enable pagination for fields that have optional fetching with the current gqlgen api because the generated resolver's method definition doesn't accept any variables from the query (in this case the variables would be the cursor, filter and sort variables).